### PR TITLE
Update front-page

### PR DESCRIPTION
**Pull request by achoum**

### CODE REVIEW

**Review by ianspektor — APPROVED** [review]
Love the more to the point approach + the more complex example 👍🏼  left some grammar/spelling nits

File: README.md [hunk 1]
```diff
@@ -6,86 +6,107 @@
...
+**Temporian** is an library for **safe**, **simple** and **effcient** preprocessing and feature engineering of temporal data in machine learning pipelines in Python. Temporian supports multivariate time-series, multivariate time-sequences, event logs, and cross-source event streams.
 
-> Temporal data is any form of data that represents a state in time. In
-> Temporian, temporal datasets contain [events](https://temporian.readthedocs.io/en/stable/user_guide/#events-and-eventsets), which consists of
-> values for one or more attributes at a given timestamp. Common
-> examples of temporal data are transaction logs, sensor signals, and
-> weather patterns. For more, see
-> [What is Temporal data](https://temporian.readthedocs.io/en/stable/user_guide/#what-is-temporal-data).
+Temporian is to [temporal data](https://temporian.readthedocs.io/en/stable/user_guide/#what-is-temporal-data) what Pandas is to tabular data.
```
[the remark this answers is not in the document]
Comment (dekked) [reply README.md:11]: Love it too :P

File: README.md [hunk 2]
```diff
@@ -6,86 +6,107 @@
...
+ and multi-index data, and both mono-source and multi-source non-synchronized
+ events.
 
+- **Protect from unwanted future leakage** 😰: Unless explicitly specified with
+`tp.leak`, features computation cannot depend on future data, thereby preventing
+ unwanted, hard-to-debug, and potentially costly future leakage.
+
+ - **Optimized for Temporal data** 🔥: Temporian's core computation is
+ implemented in C++ and optimized for temporal data. Temporian can be more than
+ 1,000x faster than non-temporal-specific libraries when operating on temporal
```
Comment (rstz) [inline README.md:26]: non-temporal-specific libraries --> off-the-shelf data processing libraries ?

File: README.md [hunk 3]
```diff
@@ -6,86 +6,107 @@
...
+ data.
+
+ - **ML library agnostic:** Temporian does not perform any machine learning
```
Comment (rstz) [inline README.md:29]: How about

**Integrates with all ML libraries:** 
Temporian does not perform any ML model training - instead it integrates seamlessly with any ML library, such as PyTorch, Scikit-Learn, Jax, or TensorFlow.

Comment (rstz) [reply README.md:29]: Or `Easy to integrate into an existing ML ecosystem` ?

File: README.md [hunk 4]
```diff
@@ -6,86 +6,107 @@
...
+Finaly, we can export the result as a Pandas dataframe for further processing or for consumption by other libraries.
+
+```python
+tp.to_pandas(aggregated_revenue)
 ```
 
-![](https://github.com/google/temporian/raw/main/docs/src/assets/frontpage_plot.png)
+![](https://github.com/google/temporian/raw/main/docs/src/assets/frontpage_pandas.png)
 
 Check the [Getting Started tutorial](https://temporian.readthedocs.io/en/stable/tutorials/getting_started/) to try it out!
```
Comment (rstz) [inline README.md:111]: try it out --> "find out more" ?

**Review by rstz — APPROVED** [review]

**Review by DonBraulio — APPROVED** [review]
Nice! I really like the new example 👌

File: README.md [hunk 5]
```diff
@@ -6,86 +6,107 @@
 ![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)
 ![publish](https://github.com/google/temporian/actions/workflows/publish.yaml/badge.svg)
 
-Temporian is an open-source Python library for preprocessing ⚡ and feature engineering 🛠 temporal data 📈 for machine learning applications 🤖. It is a library tailor-made to address the unique characteristics and complexities of time-related data, such as time-series and transactional data.
+**Temporian** is an library for **safe**, **simple** and **effcient** preprocessing and feature engineering of temporal data in machine learning pipelines in Python. Temporian supports multivariate time-series, multivariate time-sequences, event logs, and cross-source event streams.
```
Comment (dekked) [inline README.md:9]: Temporian is an library -> Temporian is **a** library

File: README.md (hunk 2 above)
Comment (dekked) [reply README.md:26]: Nitpicking, but I would change the order:

- Support most type of temporal data 📈
- Optimized for Temporal data 🔥
- ML library agnostic
- Protect from unwanted future leakage 😰

File: README.md (hunk 3 above)
Comment (dekked) [reply README.md:29]: I think we should definitely mention XGBoost here :)

File: README.md (hunk 2 above)
Comment (dekked) [reply README.md:26]: Also, on the 1,000x faster, we should definitely add a link to some benchmarks (can do later).

File: README.md [hunk 6]
```diff
@@ -6,86 +6,107 @@
...
+2023-12-04 21:21:05,STORE_31,5071
+2023-11-08 17:14:38,STORE_4,1571
+2023-11-29 21:44:46,STORE_49,6101
+2023-12-20 18:17:14,STORE_18,4499
+2023-12-15 10:55:09,STORE_2,6666
 ...
 ```
 
-The following code calculates the weekly sales for each store, visualizes the output with a plot, and exports the data to a CSV file.
+Our goal is to compute the sum of revenue for each store at 23:00 every weekday (excluding weekends).
```
Comment (dekked) [inline README.md:69]: Nitpicking, would change "23:00" by "11 pm". Americans surprisingly don't use the 24 hour clock ;)

File: README.md (hunk 2 above)
Comment (achoum) [reply README.md:26]: * apply all proposed changes.

File: README.md (hunk 3 above)
Comment (achoum) [reply README.md:29]: *Apply all proposed comments.
*Adding XGBoost and YDF :)

File: README.md [hunk 7]
```diff
@@ -6,86 +6,107 @@
...
+Consider a record of individual sale logs that contain the `timestamp`, `store`, and `revenue` of individual sales.
 
 ```shell
-$ head sales.csv
-timestamp,store,price,count
-2022-01-01,CA,27.42,61.9
-2022-01-01,TX,98.55,18.02
-2022-01-02,CA,32.74,14.93
-2022-01-15,TX,48.69,83.99
+$ !cat sales.csv
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:59]: *Removing ! since this is a "shell" box.

File: README.md (hunk 6 above)
Comment (achoum) [reply README.md:69]: Good point.
*Done

File: README.md [hunk 8]
```diff
@@ -6,86 +6,107 @@
...
 
-# Plot the result
-weekly_sum.plot()
+Then, we sum the daily revenue for each workday and each store.
 
-# Save the results
-tp.to_csv(weekly_sum, "store_sales_moving_sum.csv")
+```python
+# Aggregate revenue per store and per work day
+aggregated_revenue = sales_per_store["revenue"].moving_sum(tp.duration.days(1), sampling=work_days).rename("aggregated_revenue")
```
[the remark this answers is not in the document]
Comment (achoum) [reply README.md:95]: *Done

**Comment by achoum** [comment]
Thank all for the tomorrow review 🙂

**Pull request merged**